BD-1610: Add SMS properties for Message step callout (#4102)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/canvas_components/message_step.md

- A message is not sent because it is frequency capped
- A message is not sent because it is aborted

![Set up Messages settings for a Canvas Message component that includes the option to select your message channel and customize delivery settings.][2]{: style="max-width:75%;"}

> {% raw %}
> You can reference SMS properties in the first Message step of an action-based Canvas that is triggered by sending an SMS inbound message. For example, in the Message step, you could use `{{sms.${inbound_message_body}}}` or `{{sms.${inbound_media_urls}}}`.
> {% endraw %}

Reviewer: I'd suggest front-loading the context here, so if this situation doesn't apply to a user, they can just skip it. Also, I think "sending an SMS inbound message" isn't correct, since inbound SMS means that an end-user sent it to Braze (i.e., STOP keywords)

```suggestion
{% raw %}
If an action-based Canvas is triggered by an inbound SMS message, you can reference SMS properties in the first [Message step]({{site.baseurl}}/user_guide/engagement_tools/canvas/canvas_components/message_step/) of the Canvas. For example, in the Message step, you could use `{{sms.${inbound_message_body}}}` or `{{sms.${inbound_media_urls}}}`.
{% endraw %}
```
